cmd/server/search_handlers.go

```diff
@@ -353,15 +353,33 @@ func searchViaAddressAndName(logger log.Logger, searcher *searcher, name string,
 
 		limit, minMatch := extractSearchLimit(r), extractSearchMinMatch(r)
 
-		resp := &searchResponse{
-			RefreshedAt: searcher.lastRefreshedAt,
-		}
-
-		resp.SDNs = searcher.TopSDNs(limit, minMatch, name, keepSDN(buildFilterRequest(r.URL)))
+		sdns := searcher.TopSDNs(limit, minMatch, name, keepSDN(buildFilterRequest(r.URL)))
 
 		compares := buildAddressCompares(req)
 		filtered := searcher.FilterCountries(req.Country)
-		resp.Addresses = TopAddressesFn(limit, minMatch, filtered, multiAddressCompare(compares...))
+		addresses := TopAddressesFn(limit, minMatch, filtered, multiAddressCompare(compares...))
+
+		resp := &searchResponse{
+			SDNs:      sdns,
+			AltNames:  searcher.TopAltNames(limit, minMatch, name),
+			Addresses: addresses,
+
+			DeniedPersons:                          searcher.TopDPs(limit, minMatch, name),
+			BISEntities:                            searcher.TopBISEntities(limit, minMatch, name),
+			MilitaryEndUsers:                       searcher.TopMEUs(limit, minMatch, name),
+			SectoralSanctions:                      searcher.TopSSIs(limit, minMatch, name),
+			Unverified:                             searcher.TopUVLs(limit, minMatch, name),
+			NonproliferationSanctions:              searcher.TopISNs(limit, minMatch, name),
+			ForeignSanctionsEvaders:                searcher.TopFSEs(limit, minMatch, name),
+			PalestinianLegislativeCouncil:          searcher.TopPLCs(limit, minMatch, name),
+			CaptaList:                              searcher.TopCAPs(limit, minMatch, name),
+			ITARDebarred:                           searcher.TopDTCs(limit, minMatch, name),
+			NonSDNChineseMilitaryIndustrialComplex: searcher.TopCMICs(limit, minMatch, name),
+			NonSDNMenuBasedSanctionsList:           searcher.TopNS_MBS(limit, minMatch, name),
```

Comment:
This will perform each search in serial, so requests will take quite a long time to complete. Can we use the gatherings functions to speed this up?